pg_service_parser/pg_service_parser_wrapper.py

```diff
@@ -1,8 +1,14 @@
+import os.path
+
 import pgserviceparser
 from typing import (List,
                     Optional)
 
 
+def conf_path() -> str:
+    path = pgserviceparser.conf_path()
+    return path if os.path.exists(path) else None
```

Comment:
would be great to modify the lib to return a Path rather than a string?

Reply:
I've never used it before, but looks great.